server/calendar/calendar.js

```diff
@@ -0,0 +1,61 @@
+var gcal = require('google-calendar');
+var calendarController = require('./calendar-controller');
+
+var calendar = {
+  addCalendarGig: function (user, gigParams) {
+    var accessToken = user.accessToken;
+    var calendar_id = user.calendar_id;
+    var calendarInfo = user.gigInformation
+    gcal(accessToken).gigs.quickAdd(calendar_id, gigInformation, function(err, data) {
```

Comment:
i think you intended the second parameter to be calendarInfo, not gigInformation